src/version/version.rs

```diff
@@ -26,6 +28,13 @@ fn all(container: &Container, cfg: &Config)
         try!(b.hash(&cfg, &mut hash).map_err(|e| (format!("{:?}", b), e)));
     }
 
+    if !container.remove_all_except.is_empty() {
+        let remove_all_strings = container.remove_all_except.iter()
+            .map(|p| p.as_os_str().as_bytes())
+            .collect::<Vec<_>>();
+        hash.sequence("remove_all_except", &remove_all_strings);
```

Comment:
Make a function that accepts iterable. Or update `hash.sequence` if it doesn't break anything.